src/Student.java

```diff
@@ -0,0 +1,89 @@
+//Student class encapsulates student actors in the studying process
+public class Student extends Person{
+
+    private int tasksCompleted;
+    private static int allStudTasksCompleted;
+    private Mentor mentor;
+    private boolean allTasksCompleted;
+
+    public Student(String name, String surName, int age, Mentor mentor) {
+        super(name, surName, age);
+        this.mentor = mentor;
+        tasksCompleted = 0;
+        allTasksCompleted = false;
+    }
+
+    /* Resolves batch of tasks passed in parameters. Class instance this.tasksCompleted value must be less than
+    amount of Tasks array parameter length, otherwise it means all tasks completed so class instance
+    this.allTasksCompleted is set to true and no calls to the resolveTask method.
+    */
```

Comment:
Многострочные комментарии такого типа лучше оформлять так (это разметка по Javadoc):
```
  /**
   * Resolves batch of tasks passed in parameters. Class instance this.tasksCompleted value must be less than
   * amount of Tasks array parameter length, otherwise it means all tasks completed so class instance
   * this.allTasksCompleted is set to true and no calls to the resolveTask method.
   */
```

Reply:
resolved